### DISCUSSION TITLE
No docs for API user parameter

### DISCUSSION THREAD

**Discussion by lifeiteng** [fastapi-users/fastapi-users#1279] [Q&A] [answered]
Examples https://github.com/fastapi-users/fastapi-users/tree/master/examples/sqlalchemy-oauth
https://github.com/fastapi-users/fastapi-users/blob/master/fastapi_users/router/users.py#L38
```
    @router.get(
        "/me",
        response_model=user_schema,
        name="users:current_user",
        responses={
            status.HTTP_401_UNAUTHORIZED: {
                "description": "Missing token or inactive user.",
            },
        },
    )
    async def me(
        user: models.UP = Depends(get_current_active_user),
    ):
        return schemas.model_validate(user_schema, user)

```

<img width="1780" alt="截屏2023-08-28 20 17 18" src="https://github.com/fastapi-users/fastapi-users/assets/3107839/56dcbb37-3c25-4a6c-92d3-5f37ce66033d">

## To Reproduce
```
cd examples/sqlalchemy-oauth
uvicorn app.app:app
```

## Expected behavior

`user` should be documented, but nothing there.

Programmer will not know how to pass parameters.

**Comment by frankie567** [comment]
I'm not sure I follow you here, it looks as expected IMO.

The `GET /users/me` route is a read route returning the currently authenticated user, there is no input parameter to document.

**Reply by lifeiteng** [reply]
It need Auth Token.

```
curl -X 'GET' \
  'http://127.0.0.1:8009/users/me' \
  -H 'accept: application/json'
  -H 'Content-Type: application/json' \
  -H "Authorization: Bearer ${TOKEN}" \
```

**Reply by frankie567** [reply, marked as answer]
Yes, that's why it has the lock icon on the right. It's handled automatically by Swagger with the *Authorize* button at the top.

**Reply by lifeiteng** [reply]
Thank you.
